packages/notifi-react/lib/context/NotifiTargetContext.tsx

```diff
@@ -658,7 +658,9 @@ export const NotifiTargetContextProvider: FC<PropsWithChildren> = ({
               type: 'cta',
               message: 'Sign Wallet',
               onClick: () => {
-                // TODO: add action
+                // Note: -
+                // No action required here as this is only possible on the client side
+                // based on the connected wallet instance. We're handling this on the frontend.
```

Comment:
In my opinion, we can migrate those xmtp sign methods to `NotifiTargetContext`. this way:
1. we can simplify the component logic making it easier to follow.
2. we can reuse that method possible in the future if other components need that.